examples/chatmemberbot.py

```diff
@@ -113,7 +113,11 @@ def show_chats(update: Update, context: CallbackContext) -> None:
     update.effective_message.reply_text(text)
 
 
-def greet_chat_members(update: Update, _: CallbackContext) -> None:
+# Best practice would be to replace context with an underscore,
+# since context is an unused local variable.
+# This being an example and not having context present confusing beginners,
```

Comment:
Because this is an example and the underscore might confuse Python beginners, ... ?

Also looking at the repetition, what would you think of putting this remark in the Readme instead?

Reply:
yeah, I like that.